src/vm/vm_core.rs

```diff
@@ -2276,22 +2215,15 @@ mod tests {
             opcode: Opcode::NOp,
         };
 
-        let mem_arr = vec![(
-            MaybeRelocatable::from((0, 0)),
-            MaybeRelocatable::Int(bigint!(0x206800180018001_i64)),
-        )];
-
-        let mut vm = VirtualMachine::new(bigint!(127), Vec::new(), false);
+        let mut vm = vm!();
+        vm.memory = memory![((0, 0), 0x206800180018001_i64)];
 
-        vm.memory =
-            memory_from(mem_arr.clone(), 1).expect("Unexpected memory initialization failure");
         vm.run_context.pc = MaybeRelocatable::from((0, 0));
         vm.run_context.ap = MaybeRelocatable::from((0, 0));
         vm.run_context.fp = MaybeRelocatable::from((0, 0));
```

Comment:
This initialization can be removed, as VirtualMachine::new() already sets this values to (0,0)

Reply:
Resolved: https://github.com/lambdaclass/cleopatra_cairo/pull/378/commits/4cc1177126c96cf299bb20c3bc0b53b01b0fc3aa